src/io/flutter/devtools/DevToolsManager.java

```diff
@@ -317,6 +335,7 @@ public void onTextAvailable(@NotNull ProcessEvent event, @NotNull Key outputType
           catch (JsonSyntaxException e) {
             processHandler.destroyProcess();
           }
+          notification.expire();
```

Comment:
glad to see the notification is being closed as soon as devtools is open!

Reply:
what happens if notification.expire() is called before the notification is shown?
Will that call throw or will the subsequent call to Notification.Bus.notify show the notification?

Reply:
I tested by putting an extra `notification.expire()` statement before calling `Notification.Bus.notify` and verified that no notification is shown in that situation. Also verified that external projects where this step takes negligible time does not display the notification.